hack/lib/util.sh

```diff
@@ -325,6 +325,43 @@ kube::util::gv-to-swagger-name() {
   esac
 }
 
+
+# Fetches swagger spec from apiserver.
+# Assumed vars:
+# SWAGGER_API_PATH: Base path for swaggerapi on apiserver. Ex:
+# http://localhost:8080/swaggerapi.
+# SWAGGER_ROOT_DIR: Root dir where we want to to save the fetched spec.
+# VERSIONS: Array of group versions to include in swagger spec.
+kube::util::fetch-swagger-spec() {
```

Comment:
This function will become a lot simpler when we have swagger 2.0.
All logic to determine which paths to fetch will be gone. Just fetch /swagger.json
cc @mbohlool